docs/walkthroughs/download-artifact-to-device.adoc

```diff
@@ -0,0 +1,59 @@
+[[mobile-client-builds]]
+= Mobile Client Builds
+
+== Open Mobile Client Builds
+To download artifact you need navigate to Overview -> Mobile Clients -> <your-mobile-client>
+
+image::images/mobile-client-overview.png[openshift-overview-tab]
+
+Then go to Builds tab to open mobile client builds
+
+image::images/mobile-client-builds-tab.png[openshift-mobile-client-builds-tab]
+
+== Start Mobile Client Build
+As you see that there are not builds currently available
+When you want to trigger new build press [Start Build] button
+
+After starting build you can see notification that build was started and also
+you see that build was triggered under BUILDS section
+
+image::images/mobile-client-builds-trigger-build.png[openshift-mobile-client-builds-trigger-build]
+
+== Download Mobile Client Artifact
+You need to wait until build is completed and then press [Download] button
```

Comment:
You need to wait until build is completed and then press [Download] button -> You need to wait until the build is completed and then press [Download] button